app/retail/emails.py

```diff
@@ -920,13 +920,13 @@ def render_new_bounty_roundup(to_email):
 Hey Gitcoiners,
 </p>
 <p>
-    We’re proud to announce that this week marks the launch of our newest product, Gitcoin Quests. Quests is an enjoyable, gamified way to learn about the Web3 ecosystem, compete with your friends, earn rewards, and level up your knowledge. We’ve been working hard for the past couple months and are excited to see how you like it. <a href="https://gitcoin.co/blog/gitcoin-quests-is-now-live/">To begin your adventure, click here.</a>
+    🌱 Gitcoin Grants CLR Round 3 was a massive, massive success. With wide ranging support from across the entire Ethereum ecosystem, we were blown away with the support you provided: 1,982 unique contributions to 68 projects worth $170,000. If you'd like to take a closer look at where the record $270k ended up, take a look at our blog post. <a href="https://gitcoin.co/blog/gitcoins-q3-match/">It's living here.</a>
```

Comment:
can we pls link to this post at some point?  https://vitalik.ca/general/2019/10/24/gitcoin.html

the number one person in the entire ecosystem wrote about us.. and it's a damn good article. we should let people know to check it out.

New Copy: 

'''

Gitcoin Grants CLR Round 3 was a massive, massive success. With wide ranging support from across the entire Ethereum ecosystem, we were blown away with the support you provided: 1,982 unique contributions to 68 projects worth $170,000. In total, between individual contributions the round ended up worth $270k.
</p>
<p>
In a review of Gitcoin Grants Round 3, Ethereum Co-Founder Vitalik Buterin says "If Gitcoin Grants does continue operating regularly, and attracts and expands its pool of donations, we could be very close to seeing the first "quadratic freelancer" - someone directly "working for the public", funded by donations boosted by quadratic matching subsidies."<a href=" https://vitalik.ca/general/2019/10/24/gitcoin.html">Click here to read the article.</a>
'''

Reply:
didn't see this! so cool.